src/clarity-angular/popover/providers/dropdown.provider.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {DropdownService} from "./dropdown.service";
+
+export function clrDropdownProviderFactory(existing: DropdownService) {
```

Comment:
You can put this inline in the providers declaration if you want, a separate file is a bit overkill.